validator/tests/unit3/test_scheduler/tests.py

```diff
@@ -35,18 +35,29 @@
 
 LOGGER = logging.getLogger(__name__)
 
-
-def create_transaction(name, private_key, public_key):
+def create_transaction(name, private_key, public_key, inputs=None,
```

Comment:
It would nice to have dependencies in the function signature, so it isn't alway [].